### ISSUE TITLE
The Big One: Data Visualization Breakdown

### ISSUE THREAD

**Issue by jo-im** [SCPR/SCPRv4#1358]
AC:
- [x] The breakdown box knows when a pin is above specific layers and reports accordingly

**Comment by jo-im** [comment]
Link to Zeplin: https://app.zeplin.io/project/5afdcac3a7c15f52212a67ea/screen/5b9ad46e8adf11a16f7388e4

**Comment by jo-im** [comment]
Breakdown box now shows liquefaction and faults status of searched address. I have also fixed some issues in Safari and Internet Explorer in the data visualization view. I have merged and pushed these changes to master. 

Here is a screenshot of the data visualization page:
![Screen Shot 2018-11-26 at 3.25.47 PM.png](https://images.zenhubusercontent.com/58d7395251aa2c3057797c0d/343c7d18-cec8-42de-bf5f-75cae5f2477b)

The commit that holds these changes are here:
https://github.com/SCPR/the-big-one/commit/e7a7942e987526bc734c580d09070164dd26f5c3
